css/style.css

```diff
@@ -0,0 +1,6 @@
+
+p {
+  font-family: GothaPro;
+  font-size: 13px;
+  line-height: 16px;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5594413)